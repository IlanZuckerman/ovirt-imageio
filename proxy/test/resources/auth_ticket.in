{"transfer-ticket": "f6fe1b31-1c90-4dc3-a4b9-7b02938c8b41", "iat": 1452291246, "exp": 2452291246, "imaged-uri": "https://localhost:54322", "nbf": 1452291246}
