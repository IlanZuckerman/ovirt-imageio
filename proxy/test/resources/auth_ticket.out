eyJ2YWxpZEZyb20iOiAiMjAxNjA0MTUyMzIxNTgiLCAiY2VydGlmaWNhdGUiOiAiLS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tXG5NSUlEd3pDQ0FxdWdBd0lCQWdJQ0VBQXdEUVlKS29aSWh2Y05BUUVGQlFBd1NERUxNQWtHQTFVRUJoTUNWVk14XG5GREFTQmdOVkJBb01DMnh2WTJGc1pHOXRZV2x1TVNNd0lRWURWUVFEREJwbmNHRmtaMlYwZEM1c2IyTmhiR1J2XG5iV0ZwYmk0ek5qYzJNVEFlRncweE5URXlNRE15TVRReE1UWmFGdzB5TlRFeU1ERXlNVFF4TVRaYU1FZ3hDekFKXG5CZ05WQkFZVEFsVlRNUlF3RWdZRFZRUUtEQXRzYjJOaGJHUnZiV0ZwYmpFak1DRUdBMVVFQXd3YVozQmhaR2RsXG5kSFF1Ykc5allXeGtiMjFoYVc0dU16WTNOakV3Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLXG5Bb0lCQVFDcFV5QXVaYUg3QlVhTUFuWUFDYTQ5S29jT1QyUVpUU3U4akxGdjVJM0FEV0pRRmV3T2JSR3kySHg5XG5UdXcvNUd3Y0t1eWVGT3NxRTlyNmwwZzJ5TlpQeXRVZmRhVDNkaVF5b0xzb3NyaW9xWmhmblNLVjVBZm9TaXhuXG5xaXJKMEEyVlNqbkZIanNleHVEWTJxOWVOQ01kcndyVFBRelBHUFZsbzhtWXNvblRBOGVleXUwOUpBYzZ1QnN3XG5HL2NLbDRrYjkxTDdVMEdBaU1tdFd4QTZ4QmNTL0wxMWZ0cXFnbzlTNmRUaDBMY25WQ0ZQQVBJcG1xSWJ0RDU4XG4wbDdTdExVem1qNFNFVzJzak5tUWttajdPL1RSaUVxWlg2N1Q4NkhmRjJ3RWFUV3FCWXVFTEltbjlhSjBzY0E2XG5tMTdML1VLREpwNmhHQjhPeVJ5eWFTM1BQTHFGQWdNQkFBR2pnYll3Z2JNd0hRWURWUjBPQkJZRUZMSHhIdnVEXG43bmNhUFpOa2M3WmxHTGFvdUZEME1IRUdBMVVkSXdScU1HaUFGTEh4SHZ1RDduY2FQWk5rYzdabEdMYW91RkQwXG5vVXlrU2pCSU1Rc3dDUVlEVlFRR0V3SlZVekVVTUJJR0ExVUVDZ3dMYkc5allXeGtiMjFoYVc0eEl6QWhCZ05WXG5CQU1NR21kd1lXUm5aWFIwTG14dlkyRnNaRzl0WVdsdUxqTTJOell4Z2dJUUFEQVBCZ05WSFJNQkFmOEVCVEFEXG5BUUgvTUE0R0ExVWREd0VCL3dRRUF3SUJCakFOQmdrcWhraUc5dzBCQVFVRkFBT0NBUUVBVTR6T0ZqM1Bkc3U5XG5CRUt0K0V1bXZuc1ZEMDFXMVluejR2QzhLcDB4QkNBSUJGY0VpT1dmQ2dJSXJJbXRxRHlTNFJMbW92YzhUNGFEXG5raW5kWWxJb0YwWjZNZStrWjhmN3VIVzhPaDJCdFR6cDFjSGZ5UURXV0w1T2lVTG95UWQ2L0tKb3hNbklSSmxnXG45ejFNTWJOeUFwd2pLUTcrVWJ0cm1KVi9MV293VW9uWGhsc1BrZlBpQVhJNExTeXV4TTlWa0NZOXFCaGw0UU8rXG5tVk1OdW9KdEt0eXpicnhXbGYvRVljRHluUjl5NkI0aUhWVlRjNk5HZHU2KzN2WXdrTG1WY215L2FuWFNaWTRDXG5LWDRGckFwRCt5WGVQUmN3YXdZNjRLU29BaU1KOE1wdzBDSHN2bUdOek1GZXJOb3cyMDFwU0pqRnB3anQ4eGd0XG44ZUtrc2d0UUtRPT1cbi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS1cbiIsICJzYWx0IjogIm80NStObDJwTW9NPSIsICJ2YWxpZFRvIjogIjIwNDYwNDA4MjMyMTU4IiwgInNpZ25hdHVyZSI6ICJMSUZlQmVSVWg1SXBCTFE2NHdWYysvaXY3a3dSbkhKNW5PcC9nMVpGR0lFTy9HV0paMmE1R3VtT1lSdHJyNVdXUHpYODdxUmhjdlV3K3lsWWppYkk3MjVkcEJsRHUrY2M5UVFNOWMxaHcwOXhxTWxCNTlKRU5SbDlmTEFXUzRBWHc1bFN2bVExTHpTZTg3Rk9uQzVteUVNMkRsSlgwT0QvaTRydlluWlBJQUlEcndOcU5WM0lDNTNXRHRMdS9hMEZJUjRpOElpWXNQd01vM2p4SG1JbTExZExwU09mWUxGTUZTZnY5aE9lTXhMcEREdTJUMnBESTRoYnlBZkZqWW9GeFhWL2kzZVZMVGhlb1hiZFZDNExQMC9DajVSV3BvcWpETHhnaDh5bkF3YmVaMXFXZTJPVjRldGNiSEJ2YWJ5aGFMb2RyMHlHTEtLMGpiZlE5TWJjNlE9PSIsICJzaWduZWRGaWVsZHMiOiAiZGF0YSx2YWxpZFRvLHZhbGlkRnJvbSxzYWx0LGRpZ2VzdCIsICJkYXRhIjogIntcInRyYW5zZmVyLXRpY2tldFwiOiBcImY2ZmUxYjMxLTFjOTAtNGRjMy1hNGI5LTdiMDI5MzhjOGI0MVwiLCBcImlhdFwiOiAxNDUyMjkxMjQ2LCBcImV4cFwiOiAyNDUyMjkxMjQ2LCBcImltYWdlZC11cmlcIjogXCJodHRwczovL2xvY2FsaG9zdDo1NDMyMlwiLCBcIm5iZlwiOiAxNDUyMjkxMjQ2fVxuIiwgImRpZ2VzdCI6ICJzaGExIn0=
